docs/_data/portland-2020-schedule.yaml

```diff
@@ -98,16 +86,18 @@ talks-day2:
   - time: '1:50'
     title: Switch Speakers
   - time: '2:00'
-    slug: product-documentation-strategy-what-is-it-and-why-do-we-need-it-kay-miles
+    title: TBC
   - time: '2:30'
     title: Switch Speakers
   - time: '2:40'
-    slug: lessons-learned-in-a-year-of-docs-driven-development-jessica-parsons
+    slug: the-search-for-search-paul-logan
   - time: '3:10'
     title: Switch Speakers
   - time: '3:20'
-    slug: defying-the-status-quo-how-a-grassroots-effort-can-transform-an-organization-jodie-putrino
-  - time: '3:50'
-    title: Closing address
+    slug: globalise-the-docs-lucy-carey
   - time: '4:00'
+    slug: why-sdk-docs-matter-and-what-i-m-doing-about-it-shavindri-dissanayake-shavi
+  - time: '4:30'
+    title: Closing address
+  - time: '4:40'
```

Comment:
Hmm, did we gain an extra talk? Seems like we're not really closing early on day 2.

Reply:
Yeaah, some super tough choices *and* I figure someone will drop out.